firmware_configurations/klipper/Mellow/FLY-Gemini/cfg/printer.cfg

```diff
@@ -0,0 +1,345 @@
+## modify it if there is no other USB device
+[mcu]
+serial: /dev/ttyACM0
+
+[mcu host]
+serial: /tmp/klipper_host_mcu
+
+[printer]
+kinematics: corexy
+max_velocity: 300
+max_accel: 2000
+max_z_velocity: 5
+max_z_accel: 100
+
+
+
+#####################################################################
+# 	X/Y Stepper Settings
+#####################################################################
+
+[stepper_x]
+##	Connected to X on mcu_xye (B Motor)
+step_pin: PC13
+dir_pin: PC1
+enable_pin: !PB2
+microsteps: 16
+step_pulse_duration: 0.000004  ## This value of Fly-Gemini is recommended to be greater than or equal to 4us.
+rotation_distance: 40
+endstop_pin: PA3
+position_endstop: 120
+position_max: 120              ## VORON0 default
+homing_speed: 20
+
+##	Make sure to update below for your relevant driver (2208 or 2209)
+[tmc2209 stepper_x]
+uart_pin: PB11
+run_current: 0.800
+stealthchop_threshold: 999999
+
+
+[stepper_y]
+##	Connected to Y on mcu_xye (A Motor)
+step_pin: PC14
+dir_pin: PC4
+enable_pin: !PB6
+microsteps: 16
+step_pulse_duration: 0.000004
+rotation_distance: 40
+endstop_pin: PB1
+position_endstop: 120
+position_max: 120
+homing_speed: 20
+
+
+##	Make sure to update below for your relevant driver (2208 or 2209)
+[tmc2209 stepper_y]
+uart_pin: PB9
+run_current: 0.800
+stealthchop_threshold: 999999
+
+
+[stepper_z]
+step_pin: PC15
+dir_pin: PC5
+enable_pin: !PB5
+microsteps: 16
+step_pulse_duration: 0.0000004
+rotation_distance: 8
+endstop_pin: PB10
+position_endstop: 0.2
+position_max: 120
+
+[tmc2209 stepper_z]
+##	Make sure to update below for your relevant driver (2208 or 2209)
+uart_pin: PB8
+run_current: 0.600
+stealthchop_threshold: 999999
+
+
+
+#####################################################################
+# 	Extruder
+#####################################################################
+
+#	E0 on MCU X/Y
+[extruder]
+step_pin: PC3
+dir_pin: !PC7
+enable_pin: !PB4
+microsteps: 16
+step_pulse_duration: 0.0000004
+##	Update value below when you perform extruder calibration
+##	If you ask for 100mm of filament, but in reality it is 98mm:
+##	rotation_distance = <previous_rotation_distance> * <actual_extrude_distance> / 100
+rotation_distance: 22.23      ## VORON0 Mini AB default
+gear_ratio: 50:10             ## VORON0 Mini AB default
+nozzle_diameter: 0.400
+filament_diameter: 1.750
+heater_pin: PA0
+sensor_pin: PC0
+##	Validate the following thermistor type to make sure it is correct
+sensor_type: PT1000
+control: pid
+pid_Kp: 22.2
+pid_Ki: 1.08
+pid_Kd: 114
+min_temp: 0
+max_temp: 350
+##	E0 on MCU X/Y
+##	Make sure to update below for your relevant driver (2208 or 2209)
+[tmc2209 extruder]
+uart_pin: PB7
+run_current: 0.600
+stealthchop_threshold: 999999
+
+
+#####################################################################
+# 	Bed Heater
+#####################################################################
+[heater_bed]
+heater_pin: PA2
+sensor_pin: PC2
+sensor_type: ATC Semitec 104GT-2
```

Comment:
Comment out this sensor for the same reasons and add the same comments as for the extruder heater.